server/controllers/ArticleController.js

```diff
@@ -252,6 +253,22 @@ class ArticleController {
       );
     }
   }
+
+  /**
+   * @static
+   * @description this handles searching an article by author
+   * @param {object} req HTTP request object
+   * @param {object} res HTTP response object
+   * @returns {object} api route response
+   */
+  static async search(req, res) {
+    const { author } = req.query;
+    if (author) {
+      SearchController.byAuthor(author, res);
+    } else {
+      return response(res, 400, 'failure', 'No search parameters supplied', null, null);
```

Comment:
No need for the null values at the end

Reply:
oh, not part of my PR file changes though, but I'll check